content/seaborn/concepts/built-in-functions/terms/load-dataset/load-dataset.md

```diff
@@ -0,0 +1,64 @@
+---
+Title: 'load_dataset()'
+Description: 'Loads a sample dataset (in CSV format) from this online [repository](https://github.com/mwaskom/seaborn-data).'
+Subjects:
+  - 'Data Science'
+  - 'Data Visualization'
+Tags:
+  - 'Graphs'
+  - 'Libraries'
+  - 'Matplotlib'
+  - 'Data Structures'
+  - 'Functions'
+  - 'CSV'
+  - 'Pandas'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The `load_dataset()` function takes the name of a sample dataset (in CSV format) from this online [repository](https://github.com/mwaskom/seaborn-data) and returns a Pandas [`DataFrame`](https://www.codecademy.com/resources/docs/pandas/dataframe) object.
+
+## Syntax
+
+```pseudo
+seaborn.load_dataset(name)
+```
+
+The `name` parameter is the name of the dataset. It can be any of the datasets that exist on this online [repository](https://github.com/mwaskom/seaborn-data). It is the first parameter of the function and can be used by itself. There are, however, many other parameters that are optional or have default settings. These are usually passed as [keyword arguments](https://www.codecademy.com/resources/docs/python/functions/arguments-parameters) since using the parameter order is generally inconvenient.
+
+Some of the more significant parameters are listed here:
+
+|    Parameter Name    |         Data Type          | Usage                                                                                                                                        |
+| :------------------: | :------------------------: | -------------------------------------------------------------------------------------------------------------------------------------------- |
+|       `name`         |            str             | Name of the dataset (in CSV format) on this [repository](https://github.com/mwaskom/seaborn-data).|
+|        `cache`       |            boolean         | The default value is `True`or `1` and it allows the function to initially load the dataset from a local cache directory, after which it saves to this cache directory if a download is needed.|
+|     `data_home`      |            str             | Specifies the local directory where the cache data is stored. To view this directory, use the function `print(seaborn.get_data_home())`.|
+|       `kws`          |   dict(keys & values)      | Optional and extra keyword arguments that are passed to the pandas function `.read_csv`. This panda function reads a csv file and returns a dataframe.|
```

Comment:
```suggestion
|       `name`         |            `str`             | Name of the dataset (in CSV format) from examples repository. |
|        `cache` (optional)      |            `boolean`         | Initially loads the dataset from a local cache directory, after which it saves to this cache directory if a download is needed. |
|     `data_home`   (optional)   |            `str`             | Specifies the local directory where the cache data is stored; can be viewed with `seaborn.get_data_home()`. |
|       `kws`   (optional)       |   `dict` (keys & values)      | Extra keyword arguments that are passed to the [pandas `.read_csv()` function](https://www.codecademy.com/resources/docs/pandas/built-in-functions/read-csv). |
```

Applying these edits will most certainly mess up the spacing of this table, but don't worry about it. 😄

Before we merge, I will go in and format/lint the file(s).

Reply:
I am a bit new to this format/lint. But I am aware that they are mean for automated testing of one's work. I had node.js and also installed the packages such as yarn, prettier, markdownlint, and jest. But I have no idea how all these work, so please I would appreciate if you can help with some materials on how to learn about these and software testing xD